cms/static/js/views/active_video_upload_list.js

```diff
@@ -1,8 +1,19 @@
-define(
-    ['jquery', 'underscore', 'backbone', 'js/models/active_video_upload', 'js/views/baseview', 'js/views/active_video_upload', 'jquery.fileupload'],
-    function($, _, Backbone, ActiveVideoUpload, BaseView, ActiveVideoUploadView) {
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'js/models/active_video_upload',
+    'js/views/baseview',
+    'js/views/active_video_upload',
+    'common/js/components/views/feedback_notification',
+    'edx-ui-toolkit/js/utils/html-utils',
+    'text!templates/active-video-upload-list.underscore',
+    'jquery.fileupload'
+],
+    function($, _, Backbone, ActiveVideoUpload, BaseView, ActiveVideoUploadView, NotificationView, HtmlUtils,
+             activeVideoUploadListTemplate) {
         'use strict';
-
+        var supportedFileFormats = ['mp4', 'mov'];
```

Comment:
@mushtaqak We should have a list or dict of supported filetypes at the backend and then we can use it here.